Parse/Internal/Commands/CommandRunner/URLSession/PFURLSessionCommandRunner.m

```diff
@@ -49,11 +55,12 @@ - (instancetype)initWithDataSource:(id<PFInstallationIdentifierStoreProvider>)da
                          clientKey:(NSString *)clientKey {
     NSURLSessionConfiguration *configuration = [[self class] _urlSessionConfigurationForApplicationId:applicationId
                                                                                             clientKey:clientKey];
-    PFURLSession *session = [PFURLSession sessionWithConfiguration:configuration];
+    PFURLSession *session = [PFURLSession sessionWithConfiguration:configuration delegate:self];
```

Comment:
This is bad. Self may be re-assigned by `init`, and then we have a dead delegate value. We should think about how to improve this.